File: FountainPensNg.Server/Data/DTO/InkedUpDTO.cs
namespace FountainPensNg.Server.Data.DTO;
public record InkedUpDTO(
	 int Id,
	 DateTime InkedAt,
	 int MatchRating,
	 int FountainPenId,
	 string PenMaker,
	 string PenName,
	 int InkId,
	 string InkMaker,
	 string InkName,
	 string PenColor,
	 string InkColor,
	 string Comment);

File: fountainpensng.client/src/dtos/InkedUpDTO.ts
export interface InkedUpUploadDTO {
  id: number;
  inkedAt: Date;
  matchRating: number;
  fountainPenId: number;
  inkId: number;
  isCurrent: boolean;
  comment: string;
}

export interface InkedUpForListDTO extends InkedUpUploadDTO {
  penMaker: string;
  penName: string;
  penColor: string;
  inkMaker: string;
  inkName: string;
  inkColor: string;
  comment: string;
}
